config/manager/nephe-controller.yaml

```diff
@@ -17,6 +17,15 @@ spec:
   selector:
     control-plane: nephe-controller
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nephe-config
+  namespace: system
+data:
+  nephe-controller.conf: |
+    CloudResourcePrefix: anp
```

Comment:
This should be commented and a description on variable is required

Reply:
Solved